requirements/static/ci/linux.in

```diff
@@ -9,3 +9,5 @@ ansible
 twilio
 python-telegram-bot==12.8; python_version <= '3.5'
 python-telegram-bot>=13.7; python_version > '3.5'
+hglib
```

Comment:
Are you certain hglib and mercurial only run on linux?

Reply:
alright, found out installing mercurial into windows is a bad idea. broke python pretty bad. and was breaking i before any tests ran. set up the tests to run on the rest of the operating systems. will have to skip windows.